sources/workable/__init__.py

```diff
@@ -112,8 +112,8 @@ def _get_details(
         # A transformer functions that yield the activities and offers for each candidate.
         for sub_endpoint in DEFAULT_DETAILS["candidates"]:
             logging.info(
-                f"Loading additional data for 'candidates' from '{sub_endpoint}' in 'merge' mode."
+                f"Loading additional data for 'candidates' from '{sub_endpoint}' in 'append' mode."
             )
             yield candidates_resource | dlt.transformer(
-                name=f"candidates_{sub_endpoint}", write_disposition="merge"
+                name=f"candidates_{sub_endpoint}", write_disposition="append"
```

Comment:
hey! you are right, we missed primary_key (or merge_key) here, [candidates details response](https://workable.readme.io/reference/candidate-activities) has no id key to make it primary. I think it makes more sense to set write_disposition='replace', to avoid data duplication.

Reply:
@AstrakhantsevaAA given that candidates is incremental, it will replace all existing data with just updated ones. 
So for example,
1. Initial sync syncs 100 candidates and 10 offers
2. Candidate #20 now has an offer
3. Second pull will replace candidate offers with only a single offer for candidate #20

While merge may have duplicates, it's better than the alternative of replacing everything

Reply:
right!